cmlutils/project_entrypoint.py

```diff
@@ -264,15 +274,229 @@ def project_import_cmd(project_name):
+    )
+
+    export_username = config[USERNAME_KEY]
+    export_url = config[URL_KEY]
+    export_apiv1_key = config[API_V1_KEY]
+    output_dir = config[OUTPUT_DIR_KEY]
+    ca_path = config[CA_PATH_KEY]
+
+    export_output_dir = get_absolute_path(output_dir)
+    export_ca_path = get_absolute_path(ca_path)
+
+    log_filedir = os.path.join(output_dir, project_name, "logs")
+    _configure_project_command_logging(log_filedir, project_name)
+    logging.info("Started Verifying project: %s", project_name)
+    try:
+        # Get username of the creator of project - This is required so that admins can also migrate the project
+        pobj = ProjectExporter(
+            host=export_url,
+            username=export_username,
+            project_name=project_name,
+            api_key=export_apiv1_key,
+            top_level_dir=export_output_dir,
+            ca_path=export_ca_path,
+            project_slug=project_name,
+            owner_type="",
+        )
+        export_creator_username, export_project_slug, export_owner_type = pobj.get_creator_username()
+        if export_creator_username is None:
+            logging.error(
+                "Validation error: Cannot find project - %s under username %s",
+                project_name,
+                export_username,
+            )
+            raise RuntimeError("Validation error")
+        logging.info("Begin validating export project")
+        validators = initialize_export_validators(
+            host=export_url,
+            username=export_creator_username,
+            project_name=project_name,
+            top_level_directory=export_output_dir,
+            apiv1_key=export_apiv1_key,
+            ca_path=export_ca_path,
+            project_slug=export_project_slug,
+        )
+        for v in validators:
+            validation_response = v.validate()
+            if validation_response.validation_status == ValidationResponseStatus.FAILED:
+                logging.error(
+                    "Validation error: %s",
+                    project_name,
+                    validation_response.validation_msg,
+                )
+                raise RuntimeError(
+                    "validation error", validation_response.validation_msg
+                )
+        logging.info(
+            "Finished validating export verification validations for project %s.", project_name
+        )
+        pexport = ProjectExporter(
+            host=export_url,
+            username=export_creator_username,
+            project_name=project_name,
+            api_key=export_apiv1_key,
+            top_level_dir=export_output_dir,
+            ca_path=export_ca_path,
+            project_slug=export_project_slug,
+            owner_type=export_owner_type,
+        )
+        exported_proj_data, exported_proj_list, exported_model_data, exported_model_list,  exported_app_data, exported_app_list, exported_job_data, exported_job_list = \
+            pexport.collect_export_project_data()
```

Comment:
Run black and isort formatting tools and raise the PR again.
https://github.com/cloudera/cmlutils#development-guidelines